tests/phpunit/util/wpml-pb-test-case2.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+abstract class WPML_PB_TestCase2 extends OTGS_TestCase {
```

Comment:
@adrianosferreira, as this class is used in other page builder packages, I think it would be a good move to have it in the main `wpml-page-builders` (as this package is a dev dependency of specific PB packages). You can do this in another task.